style.css

```diff
@@ -1,74 +1,745 @@
-/* @import url("https://fonts.googleapis.com/css?family=Inter&display=swap"); */
+@import url("https://fonts.googleapis.com/css?family=Nanum+Gothic&display=swap");
```

Comment:
- [ ] Good job importing the fonts, it would be great if you used the template font (Cocogoose) as this is the font from the template.

Reply:
I can't seem to find a free non-trial/demo version of Cocogoose. When using the trial version it doesn't resemble a bit to the template. I used the Nanum Gothic, again, because it is on the live site at http://summit.cckorea.org/main.html.
![2022-04-15](https://user-images.githubusercontent.com/44879977/163524132-4596569c-0fb9-4e0e-adee-078c89e859a2.png)

Reply:
Hi @acolombo1, you can use the trial version, here's a link you can use: https://fonts.cdnfonts.com/css/cocogoose